pkg/kiosk/aws_login.go

```diff
@@ -46,7 +47,16 @@ func GrafanaKioskAWSLogin(cfg *Config, messages chan string) {
 	// Give browser time to load next page (this can be prone to failure, explore different options vs sleeping)
 	time.Sleep(2000 * time.Millisecond)
 
+	headers := make(map[string]interface{})
+	if len(cfg.BasicAuth.Username) != 0 && len(cfg.BasicAuth.Password) != 0 {
```

Comment:
same for this file, this is for aws login, and doesn't use basic auth, maybe this was not intended?

Reply:
see the following comment https://github.com/grafana/grafana-kiosk/pull/144#discussion_r1776464284
but if u think it's not needed then I'll just remove it.